PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCResults.ps1

```diff
@@ -46,28 +46,57 @@ Function Get-JCResults
 
         # Attempt initial call and collect first page of results
         try {
-            $response = Invoke-WebRequest -Method GET -Uri $limitURL -Headers $hdrs -UserAgent:(Get-JCUserAgent)
+            if ($body) {
+                $ProgressPreference = "SilentlyContinue"
+                $response = Invoke-WebRequest -Method $method -Body $body -Uri $limitURL -Headers $hdrs -UserAgent:(Get-JCUserAgent)
+            }
+            else {
+                $ProgressPreference = "SilentlyContinue"
+                $response = Invoke-WebRequest -Method $method -Uri $limitURL -Headers $hdrs -UserAgent:(Get-JCUserAgent)
+            }
+            
         }
         catch {
             throw $_
         }
 
         # Find total amount of results
-        $totalCount = $response.Headers."x-total-count"
-        $totalCount = [int]$totalCount.Trim()
-        Write-Debug "total count: $totalCount"
-
+        if ($null -eq $response.Headers."x-total-count") {
```

Comment:
This if/else statement is the difference between /API/Search and /API/systemUsers endpoints (just as an example right?) We should add a comment here.

Reply:
Agreed, I thought I had added that comment but might have accidentally deleted it at some point, will re-add